github-workflow/aks-deploy.yaml

```diff
@@ -48,6 +48,7 @@ jobs:
           az acr import --source docker.io/library/memcached:1.5.20 -n $ACR_NAME_BU0001A0042 --force
           az acr import --source docker.io/fluxcd/flux:1.19.0 -n $ACR_NAME_BU0001A0042 --force
           az acr import --source docker.io/weaveworks/kured:1.4.0 -n $ACR_NAME_BU0001A0042 --force
+          az acr import --source docker.io/library/traefik:2.2.1 -n $ACR_NAME_BU0001A0042 --force
```

Comment:
Where do you actually use these?  I don't see any `images:` directives in your kustomization files (or `sed` usage) that replace the public container registry usage with these imported images.

Reply:
I must said this is technical debt at this point. Do you want to work on this? Can we create  AzDO for this instead?

Reply:
Go ahead and create a work item for this, and maybe we can get some cycles in the next sprint to address.  Sucks to deploy ACR and never actually use it anywhere or have any instructions on how to do it (other than those buried comments inthe manifest files)

Reply:
done | it is #399511

Reply:
Cool, thanks.